graphql/schema.graphql

```diff
@@ -2,8 +2,6 @@ directive @withAuthMetrics on FIELD_DEFINITION
 
 directive @withOwnerId on FIELD_DEFINITION
 
-directive @monitorOrigin on FIELD_DEFINITION
```

Comment:
The [build is failing](https://master--vtex.myvtex.com/admin/app/builder-log/build-log-viewer/d790f8f31ec44f93867115ded51dbeb6) because we also need to remove the references here:
https://github.com/vtex-apps/store-graphql/blob/8fcecbc42409dd95629ba07555a4ff2b90fe3fda/node/directives/index.ts#L7-L16